core/domain/exp_domain.py

```diff
@@ -3553,9 +3554,41 @@ def _convert_states_v23_dict_to_v24_dict(cls, states_dict):
                 state_dict, html_validation_service.convert_to_ckeditor)
         return states_dict
 
+    @classmethod
+    def _convert_states_v24_dict_to_v25_dict(cls, exp_id, states_dict):
+        """Converts from version 24 to 25. Version 25 adds the dimensions of
+        images in the oppia-noninteractive-image tags.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+            exp_id: str. ID of the exploration.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        for key, state_dict in states_dict.iteritems():
+            add_dimensions_to_noninteractive_image_tag = functools.partial(
+                html_validation_service.add_dimensions_to_noninteractive_image_tag, # pylint: disable=line-too-long
```

Comment:
html_validation_service.add_dimensions_to_image_tags

(and change the varname in the line above accordingly)

Reply:
Done